content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -195,4 +195,27 @@ openssl command is expected.
     $ kubectl delete ns foo istio-system
     {{< /text >}}
 
-*   To remove the Istio components: follow the [uninstall instructions](/docs/setup/getting-started/#uninstall) to remove.
+*   Remove the authentication policy for the foo namespace:
+
+    {{< text bash >}}
+    $ kubectl delete peerauthentication -n foo default
+    {{< /text >}}
+
+*   Remove the sample and httpbin application:
+
```

Comment:
It should be something like : Remove the sample applications `sleep` and httpbin`: